src/js/dom.js

```diff
@@ -26,10 +26,30 @@ export function throttleWhenTyping(cb, timeout) {
         cb(form);
     }
 
-    $(document).on('change keydown', '.js-comment-field', function() {
+    function handleChange() {
         if (timer) return;
 
-        const form = $(this).closest('form').get(0);
+        const form = closestParent(this, 'form');
         timer = setTimeout(() => success(form), timeout);
+    }
+
+    live('change', '.js-comment-field', handleChange);
+    live('keydown', '.js-comment-field', handleChange);
+}
+
+function live(eventType, selector, cb) {
+    document.addEventListener(eventType, function(event) {
+        const el = closestParent(event.target, selector);
+        if (el) {
+            cb.call(el, event);
+        }
     });
 }
+
+function closestParent(parent, selector) {
```

Comment:
[`closest` was added to Chrome recently](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest) - Let's go with that